src/main/steps/edge-case/dashboard/template.njk

```diff
@@ -1,5 +1,6 @@
 {% extends "common/template.njk" %}
 
 {% block page_content %}
-<a href="/respondent/task-list" class="govuk-link">Respondent</a>
```

Comment:
@hmcts-abduldaim Lets not remove any respondent existing links please.
We just need to add new changes for c100 without altering respondent journey.